extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -141,14 +147,19 @@ export class AmpAudio extends AMP.BaseElement {
       audio
     );
 
-    applyFillContent(audio);
```

Comment:
Don't we want this still? It styles the audio element with block and a few other things I think we still want. Though I do see we're overriding the width on line 122/128.

but if it is no longer giving us styles we care about, I'm also fine removing it. I just don't have enough historical context to understand why it was here in the first place.

Reply:
I don't think we should change this in this PR.